cfme/fixtures/candu.py

```diff
@@ -0,0 +1,31 @@
+"""
+Fixtures for Capacity and Utilization
+"""
+import pytest
+
+
+@pytest.yield_fixture(scope="module")
+def enable_candu(appliance):
+    candu = appliance.collections.candus
+    server_settings = appliance.server.settings
+    original_roles = server_settings.server_roles_db
+
+    server_settings.enable_server_roles(
+        'ems_metrics_coordinator',
+        'ems_metrics_collector',
+        'ems_metrics_processor'
+    )
+    server_settings.disable_server_roles(
+        'automate',
+        'smartstate'
+    )
+    candu.enable_all()
+
+    # ToDo: Its temporary wait for the metric collection.
+    # It should replace by wait_for with proper condition depends on the provider.
+    import time
+    time.sleep(900)
```

Comment:
**REQUIRED**
@digitronik I've read your explanation here https://github.com/ManageIQ/integration_tests/pull/7041#discussion_r180976007, but still this doesn't look good. Now our test will run 15 minutes longer always. We have `wait_candu_data_available` https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/vm.py#L539. Please consider to apply it in your PR.

Reply:
Thanks, @quarckster added changes as per your suggestions. 5.8 lack of appliance but its working fine locally.